carInspectionManagment/ConnecterLayer/CreateInspectionResponseConsumer.cs

```diff
@@ -11,10 +13,24 @@ namespace ConncterLayer
 
     public class CreateInspectionResponseConsumer : IConsumer<CreateInspectionResponse>
```

Comment:
plz use clear naming for consumer you can use ConsumeInspectionFromCacheConsumer or FetcheInspectionFromCacheConsumer